spyder/plugins/outlineexplorer/api.py

```diff
@@ -22,14 +22,19 @@ def handle_go_to(name, line, text):
 
 outlineexplorer.edit_goto.connect(handle_go_to)
 """
+from qtpy.QtCore import Signal, QObject
 
 
-class OutlineExplorerProxy(object):
+class OutlineExplorerProxy(QObject):
     """
     Proxy class between editors and OutlineExplorerWidget.
     """
 
+    sig_cursor_position_changed = Signal(int, int)
+    sig_outline_explorer_changed = Signal()
```

Comment:
What is `outline_explorer_changed` ? the user selected something on the Outline?

Reply:
> The syntax highlighter created a new `oedata` or deleted one. So the user typed something in the editor.
>Maybe it should be called `sig_outline_explorer_data_changed`?

Definitely :-) `sig_outline_explorer_data_changed` 👍